File: Java Language/Java Language/Basic.java
// java - Basic
import java.util.Scanner;
class coding
{
    public static void main(String[] args)
    {
        // printing
        // println() for new line otherwise print() 
        System.out.print("HELLO");

        int number;     // int variable as sample 
        Scanner sc = new Scanner (System.in);       // initialise scanner 
        
        System.out.println("ENTER A NUMBER");     // ask for integer input

        // store input in variable
        number = sc.nextInt();

        System.out.println(number);

        // data types :
        // int ,char ,float ,double ,long ,boolean 
        // string ,array

        int i = 10;
        long l = 2.05;   
        float f = 3.14F;  
        double d = 100.25D;

        String name;
        int[] array;

        // operator :
        // unary ,binary ,trinary
        // arithamatic ,logical ,assignment ,bitwise ,relational
        // operator works same in all language

        // type casting 
        // double to int
        double num = 10;
        int num2 = (int)num;
    }
}

File: CPP/CPP Basics/Basic.cpp
// cpp - Basic
// include header file
#include <iostream>
// using standard library function
using namespace std;
// main function
int main()
{
    // output stream with endline
    cout << "Hello World" << endl;

    // input stream
    int a;
    cin >> a;
    cout << a << endl;

    // c langage function / printf anf scanf
    printf("welcome");
    scanf("%d \n", &a);
    printf("%d \n", a);

    // data types
    // int ,short ,long ,double ,float ,array
    // char ,string
    int i;
    short int si;
    long int li;
    float f;
    int array[2];  // 1D array
    int arr[2][2]; // 2D array
    char c;
    string s;

    // constant data
    const int num = 10;

    // pointer - stores address of variable
    int *p = NULL;
    *p = num;
    void *ptr = NULL;
    ptr = (void *)num;

    // operator :
    // unary ,binary ,trinary, arithamatic ,logical ,assignment ,bitwise ,relational
    // operator works same in all language
}